NovaPushNotificationServiceExtension/Handlers/Multisig/CommonMultisigHandler.swift

```diff
@@ -102,44 +100,77 @@ extension CommonMultisigHandler {
 // MARK: - Private
 
 private extension CommonMultisigHandler {
-    func createNotificationContentWrapper(
-        wallets: [Web3Alert.LocalWallet],
+    func convertAddress(
+        _ address: String,
+        for chain: ChainModel
+    ) -> AccountAddress {
+        (
+            try? address
+                .toChainAccountIdOrSubstrateGeneric(using: chain.chainFormat)
```

Comment:
is it important to require that the address matches chain/concrete prefix? Also does evm addresses work here? Probably just use `toAccountId()`